lib/generators/rolemodel/mailers/templates/app/mailers/staging_mailer_interceptor.rb

```diff
@@ -0,0 +1,8 @@
+class StagingMailerInterceptor
+  def self.delivering_email(message)
+    whitelisted_emails = ENV['WHITELISTED_EMAILS'].split(',').map(&:strip)
+    message.to = message.to & whitelisted_emails
+    message.subject = "STAGING - #{message.subject}"
```

Comment:
Might want this also be able to be `REVIEW_APP`. I think that could be pulled from the ENV